partition/roles/gitlabci-runner/tasks/main.yaml

```diff
@@ -0,0 +1,62 @@
+---
+- name: check mandatory variables for this role are set
+  assert:
+    fail_msg: "not all mandatory variables given, check role documentation"
+    quiet: yes
+    that:
+      - gitlab_runner_url is not none
+      - gitlab_runner_token is not none
+      - gitlab_runner_description is not none
+      - gitlab_runner_tags is not none
+      - gitlab_runner_hostpath is not none
+
+- name: install docker as a prerequisite to gitlabci
+  apt:
+    name:
+    - docker.io
```

Comment:
Use a dedicated docker role to do this, the role name does not imply this dependency. There are a good roles in ansible-galaxy to do this.